udata/core/discussions/api.py

```diff
@@ -198,6 +212,13 @@ def get(self):
         discussions = Discussion.objects
         if args["for"]:
             discussions = discussions.generic_in(subject=args["for"])
+        if args["org"]:
+            org = Organization.objects.get(id=ObjectId(args["org"]))
+            reuses = Reuse.objects(organization=org).only("id")
+            datasets = Dataset.objects(organization=org).only("id")
+            dataservices = Dataservice.objects(organization=org).only("id")
+            subjects = list(reuses) + list(datasets) + list(dataservices)
+            discussions = discussions(subject__in=subjects).order_by("-created")
```

Comment:
I don't think we want to apply order here?
```suggestion
            discussions = discussions(subject__in=subjects)
```